docker/README.md

```diff
@@ -1,86 +1,84 @@
 Quickstart for localhost development
 ====================================
 
-Install docker: https://docs.docker.com/engine/installation/
+Install [docker](https://docs.docker.com/engine/installation/) and [docker-compose](https://docs.docker.com/compose/install/).
 
-Following steps will bring up the docker container running temporal server
-along with all its dependencies (cassandra, statsd, graphite). Exposes temporal
-frontend on port 7233 and grafana metrics frontend on port 8080.
+Following steps will bring up the docker container running Temporal server
+along with all its dependencies (cassandra, statsd, graphite). Exposes Temporal
+frontend on port `7233` and grafana metrics frontend on port `8080`.
 
+```bash
+$ cd docker
+$ docker-compose up
 ```
-cd $GOPATH/src/github.com/temporalio/temporal/docker
-docker-compose up
-```
-
-View metrics at localhost:8080/dashboard    
-View Cadence-Web at localhost:8088  
-Use Temporal-CLI with `docker run --network=host --rm temporalio/tctl:latest`
 
-For example to register new namespace 'test-namespace' with 1 retention day
-`docker run --network=host --rm temporalio/tctl:latest --ns test-namespace namespace register -rd 1`
+View metrics at `localhost:8080/dashboard`.
+View Temporal web UI at `localhost:8088`.
+Use Temporal CLI with `docker run --network=host --rm temporalio/tctl:latest`.
 
+For example to register new namespace `test-namespace` with 1 retention day:
+```bash
+docker run --network=host --rm temporalio/tctl:latest --ns test-namespace namespace register -rd 1`
+```
 
 Using a pre-built image
 -----------------------
 With every tagged release of the temporal server, there is also a corresponding
 docker image that's uploaded to docker hub. In addition, the release will also
 contain a **docker.tar.gz** file (docker-compose startup scripts). 
-Go [here](https://github.com/temporalio/temporal/releases/latest) to download a latest **docker.tar.gz** 
+[Download](https://github.com/temporalio/temporal/releases/latest) a latest **docker.tar.gz**.
```

Comment:
Add command line way to download the release:

curl -L https://github.com/temporalio/temporal/releases/latest/download/docker.tar.gz|tar -xz

We have a bunch of places where we give curl command to download the docker-compose from master. I think all of them should be replaced with the above command.